governance/handler.go

```diff
@@ -252,10 +286,38 @@ func checkBigInt(k string, v interface{}) bool {
 	return false
 }
 
-func checkAddress(k string, v interface{}) bool {
+func checkAddressOrListOfUniqueAddresses(k string, v interface{}) bool {
+	if _, ok := v.(common.Address); ok {
+		return true
+	}
+	if _, ok := v.([]common.Address); !ok {
+		return false
+	}
+
+	// there should not be duplicated addresses, if value contains multiple addresses
+	addressExists := make(map[common.Address]bool)
+	for _, address := range v.([]common.Address) {
+		if _, ok := addressExists[address]; ok {
+			return false
+		} else {
+			addressExists[address] = true
+		}
+	}
 	return true
 }
 
+func isEqualValue(k int, v1 interface{}, v2 interface{}) bool {
+	if reflect.TypeOf(v1) != reflect.TypeOf(v2) {
+		return false
+	}
+
+	if GovernanceItems[k].t != addressT || reflect.TypeOf(v1) == addressT {
+		return v1 == v2
+	}
+
+	return reflect.DeepEqual(v1.([]common.Address), v2.([]common.Address))
```

Comment:
I think the following code is more intuitive.

```suggestion
	if GovernanceItems[k].t == addressT && reflect.TypeOf(v1) != addressT {
		return reflect.DeepEqual(v1.([]common.Address), v2.([]common.Address))
	}

	return v1 == v2
```

Reply:
Thanks, it is reflected at https://github.com/klaytn/klaytn/pull/1110/commits/81771918b0d94ab90c7f28fc8df92912dbe6280d